README.md

```diff
@@ -17,9 +17,9 @@ Parameterized-test for Swift. (with XCTest)
 
 SwiftParamTest supports two way of code-style dependent on Swift version.
 
-### Function builders API (recommended)
+### Result builders API (recommended)
 
-I recommend this when you use Swift 5.1 or later (because this API uses Function builders).
+I recommend this API when you use Swift 5.4 or later (because this API uses Result builders).
```

Comment:
```suggestion
I recommend this API when you use Swift 5.1 or later (because this API uses Result builders).
```

Reply:
Result Builder is *not* available in Swift5.1. So This suggestion needs to be revised.

Reply:
Correct description in 8099dccbda3c8705c3eae01ee306c039a7732d2d